default.yml

```diff
@@ -58,6 +58,10 @@ Style/NestedModifier:
 Style/SoleNestedConditional:
   Enabled: true
 
+# https://docs.rubocop.org/rubocop/cops_layout.html#layoutaccessmodifierindentation
+Layout/AccessModifierIndentation:
+  EnforcedStyle: outdent
+
```

Comment:
This is a great opportunity to test out the drafted flow, thanks! 🙌🏻

The proposal is to add [`Severity: info`](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Severity) to restricting changes, allowing some time for teams to adopt new restricting agreements on our style. Some months later, we can remove the `Severity: info` thingy to make the change mandatory.

We need to settle things and communicate changes to the teams before moving forward with the PR, but it would be *great* to have your early feedback on the proposal ❤️

Reply:
i don't think that we need to make this severity: info, because rubocop checks are performed only on the changed files, thus the impact is scoped to the code that you are workin on.